tests/test_utils.py

```diff
@@ -64,3 +66,9 @@ def assert_log(simulation_object):
         assert_log(sub_process)
 
     return new_log
+
+
+def test_notebook_path():
+    """test if we can find the notebook path"""
+    notebook_path = openclsim.utils.find_notebook_path()
+    assert "notebook" in str(notebook_path), "notebook should be in the notebook path"
```

Comment:
I guess we are not running the notebooks in the test yet?

Reply:
The notebooks can be run as separate tests. That's another task. In OpenTNSim we do that in [circleci](https://github.com/TUDelft-CITG/OpenTNSim/blob/master/.circleci/config.yml#L23)